client/lib/domains/resolve-domain-status.js

```diff
@@ -252,18 +345,51 @@ export function resolveDomainStatus(
 					status: translate( 'Expiring soon' ),
 					icon: 'info',
 					listStatusText: expiresMessage,
+					noticeText: expiresMessage,
 					listStatusClass: 'warning',
 					listStatusWeight: 800,
 				};
 			}
 
 			if ( isRecentlyRegistered( domain.registrationDate ) ) {
+				let noticeText;
+				if ( domain.isPrimary ) {
+					noticeText = translate(
+						'We are setting up your domain. It should start working immediately, but may be unreliable during the first 30 minutes. If you are unable to access your site at %(domainName)s, try setting the primary domain to a domain you know is working. {{learnMore}}Learn more{{/learnMore}} about setting the primary domain, or try {{try}}%(domainName)s{{/try}} now.',
+						{
+							args: {
+								domainName: domain.name,
+							},
+							components: {
+								learnMore: (
+									<a href={ SETTING_PRIMARY_DOMAIN } rel="noopener noreferrer" target="_blank" />
+								),
+								try: (
+									<a href={ `http://${ domain.name }` } rel="noopener noreferrer" target="_blank" />
+								),
+							},
+						}
+					);
+				} else {
+					noticeText = translate(
+						'We are setting up your domain. It should start working immediately, but may be unreliable during the first 30 minutes. {{learnMore}}Learn more{{/learnMore}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'We are setting up your new domains for you. They should start working immediately, but may be unreliable during the first 30 minutes. {{domainsLink}}Learn more{{/domainsLink}}.' )` *ES Score: 7*